plugins/cloudinary/jest.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  roots: ['<rootDir>'],
+  testRegex: '.*\\.(test|spec).(ts|tsx|js)$',
+  coverageDirectory: 'coverage/',
+  transform: {
+    '^.+\\.tsx?$': 'ts-jest'
+  },
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json', 'node'],
+  testEnvironment: 'jsdom',
+  setupFilesAfterEnv: ['<rootDir>/src/setupEnzyme.ts'],
+  coverageDirectory: 'coverage/',
+  collectCoverage: true,
+  coverageThreshold: {
+    global: {
+      branches: 80,
+      functions: 80,
+      lines: 80,
+      statements: 80
+    }
+  },
+  collectCoverageFrom: ['src/**']
+}
```

Comment:
I thought it was a good idea to get the jest configuration out of the package.json file.